.travis.yml

```
@@ -16,6 +16,7 @@ before_script:
 script:
   - bundle exec rails db:create
   - bundle exec rails db:schema:load
+  - bundle exec rubocop app spec lib Gemfile
```

```suggestion
  - bundle exec rubocop app spec lib Gemfile config
```

Or can we run `bundle exec rubocop` and get the default directories? Or was this done to match the existing behaviour and then we can fix it later (I'm guessing there might be linting issues in the `config` directory?).

---

When I did this, I didn't have the idea to do the `rubocop-govuk` changes yet (it was only after several goes at this that it got tedious enough). And this was done to match Jenkins (even though it's not Jenkins, but consistency).

---

This will get a RuboCop update in due course and in doing that we can remove the directories from this line.